toolkit/governed-map/primitives/src/lib.rs

```diff
@@ -403,6 +423,10 @@ sp_api::decl_runtime_apis! {
 	#[api_version(1)]
 	pub trait GovernedMapIDPApi
 	{
+		/// Returns initialization state of the pallet
+		fn is_initialized() -> bool;
+		/// Returns an iterator over all mappings currently stored in the pallet
```

Comment:
Not an iterator but BTreeMap, which I like more btw. Being strict at output is a good thing.